spyder/widgets/tabs.py

```diff
@@ -48,6 +48,12 @@ def __init__(self, parent, ancestor):
         self.setAcceptDrops(True)
         self.setUsesScrollButtons(True)
 
+        # Tab name editor
+        self.rename_tabs = rename_tabs
+        if self.rename_tabs is True:
+            # Creates tab name editor
+            self.tab_name_editor = EditTabNamePopup(self)

```

Comment:
This variable is only getting created if true, so maybe we should define a 

`self.tab_name_editor = None` above